src/Tribe/Amalgamator.php

```diff
@@ -258,8 +258,8 @@ private function delete_posts( $post_ids ) {
 	 */
 	public static function migration_button( $text = '' ) {
 		$text     = $text ? $text : __( 'Merge Duplicates', 'tribe-events-calendar' );
-		$html     = '<a href="%s" class="button">%s</a>';
 		$settings = Tribe__Events__Settings::instance();
+		
```

Comment:
Super minor: codesniffer is flagging up the unnecessary whitespace here (two tabs).